### PR TITLE
[Doc Improvement][Remove duplicate SDK references from articles]

### PR DESCRIPTION
**Pull request by v-ypalikila**
There were duplicate link for SDK references in multiple articles. Created the PR to remove all the duplicate links.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 93 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2474bae4-307e-49a6-af69-b3eecb918f90) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fbot-features.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/456ec3a5-ca49-46e9-aad2-a9d293051541) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 94 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01dd8d95-477a-4196-a616-5e9983ec31f5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-messages-with-rsc.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a17979d-cdb3-4b93-a925-b87aa10d9720) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 92 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6dd30da5-7797-4f9b-a8bd-3dd12e2c7aab) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/send-proactive-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fsend-proactive-messages.md) | 95 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4777c6f0-b623-450a-9719-d6a73a6892f2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fsubscribe-to-conversation-events.md) | 95 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89a8e731-1909-4a93-9fac-df0d33cd93be) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/create-a-bot-commands-menu.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fcreate-a-bot-commands-menu.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/43fcf97c-82b7-429a-a500-a514029c1507) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/get-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fget-teams-context.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c76eca23-e7e1-49d7-8b30-2de08d1fb620) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/update-and-delete-bot-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fbots%2Fhow-to%2Fupdate-and-delete-bot-messages.md) | 94 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50dbf4dd-313b-469f-887a-434c53a9220d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fgraph-api%2Fproactive-bots-and-messages%2Fgraph-proactive-bots-and-messages.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/76b02096-ff46-4c34-bb07-303aee6cf4ac) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Frespond-to-search.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be9480d0-0b9d-46fe-a16e-c0421db05312) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/what-are-messaging-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Fmessaging-extensions%2Fwhat-are-messaging-extensions.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b500171-45e1-4c25-8f38-1caa92e3d8ec) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d91ea5e1e266c1c77a36bc0975ee0eb372e6961a/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 92 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1dca968c-94be-407a-b90f-5858e84e9026) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md [hunk 1]
```diff
@@ -65,8 +65,6 @@ The following table includes the query parameters:
 ### Example
 
 # [C#](#tab/dotnet)
-* [SDK reference](/dotnet/api/microsoft.bot.builder.teams.teamsinfo.getmeetingparticipantasync?view=botbuilder-dotnet-stable&preserve-view=true)  
-* [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/meetings-context-app/csharp/MeetingContextApp/Bots/MeetingContextBot.cs#L33)
 
 * [SDK reference](/dotnet/api/microsoft.bot.builder.teams.teamsinfo.getmeetingparticipantasync?view=botbuilder-dotnet-stable&preserve-view=true)  
 * [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/meetings-context-app/csharp/MeetingContextApp/Bots/MeetingContextBot.cs#L19)
```
Comment (ChetanSharma-msft) [inline msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md:70]: Replace it with below URL:
* [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/meetings-context-app/csharp/MeetingContextApp/Bots/MeetingContextBot.cs#L33)

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md [hunk 2]
```diff
@@ -176,8 +179,6 @@ The Bot Framework SDK provides helper methods and objects to create mentions.
 The following code shows an example of adding mentions to your messages:
 
 # [C#](#tab/dotnet)
-* [SDK reference](/dotnet/api/microsoft.bot.schema.mention?view=botbuilder-dotnet-stable&preserve-view=true)
-* [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs#L300)
 
 * [SDK reference](/dotnet/api/microsoft.bot.builder.activityhandler.onmessageactivityasync?view=botbuilder-dotnet-stable&preserve-view=true)
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md:183]: Remove these references and revert the above once.

File: msteams-platform/bots/how-to/conversations/conversation-messages.md [hunk 3]
```diff
@@ -395,10 +404,9 @@ If you want to generate an arbitrary notification without sending a message to t
 The following code shows an example of adding notifications to your message:
 
 # [C#](#tab/dotnet)
-* [SDK reference](/dotnet/api/microsoft.bot.builder.teams.teamsactivityextensions.teamsnotifyuser?view=botbuilder-dotnet-stable#microsoft-bot-builder-teams-teamsactivityextensions-teamsnotifyuser(microsoft-bot-schema-iactivity)&preserve-view=true)
-* [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-proactive-messaging/csharp/proactive-cmd/Program.cs#L178)
 
 * [SDK reference](/dotnet/api/microsoft.bot.builder.activityhandler.onmessageactivityasync?view=botbuilder-dotnet-stable&preserve-view=true)
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/conversations/conversation-messages.md:408]: Remove these and revert the above once.

File: msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md [hunk 4]
```diff
@@ -1501,10 +1509,9 @@ In this example, the `conversation.id` of the `conversationUpdate` and `installa
 > The selected channel id is only set on `installationUpdate` *add* events that are sent when an app is installed into a team.
 
 # [C#](#tab/dotnet)
-* [SDK reference](/dotnet/api/microsoft.bot.builder.activityhandler.oninstallationupdateactivityasync?view=botbuilder-dotnet-stable#microsoft-bot-builder-activityhandler-oninstallationupdateactivityasync(microsoft-bot-builder-iturncontext((microsoft-bot-schema-iinstallationupdateactivity))-system-threading-cancellationtoken)&preserve-view=true)
-* [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs#L70-L80)
 
 * [SDK reference](/dotnet/api/microsoft.bot.builder.cloudadapterbase.continueconversationasync?view=botbuilder-dotnet-stable#microsoft-bot-builder-cloudadapterbase-continueconversationasync(system-string-microsoft-bot-schema-activity-microsoft-bot-builder-botcallbackhandler-system-threading-cancellationtoken)&preserve-view=true)
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:1513]: Please remove these and revert the above once.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md:70]: updated.

File: msteams-platform/bots/how-to/conversations/conversation-messages.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/conversations/conversation-messages.md:408]: updated.

File: msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md (hunk 4 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:1513]: Updated.

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md:183]: Updated.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**